airbyte-integrations/connectors/source-google-search-console/integration_tests/expected_records.jsonl

```diff
@@ -0,0 +1,46 @@
+{"stream": "sites", "data": {"permissionLevel": "siteOwner"}, "emitted_at": 1674600072721}
+{"stream": "sites", "data": {"permissionLevel": "siteOwner"}, "emitted_at": 1674600073155}
+{"stream": "sitemaps", "data": {"lastSubmitted": "2021-09-10T23:02:22.258Z", "isPending": false, "isSitemapsIndex": false, "type": "sitemap", "warnings": "0", "errors": "0", "contents": [{"type": "web", "submitted": "5690", "indexed": "0"}]}, "emitted_at": 1674600073428}
+{"stream": "sitemaps", "data": {"lastSubmitted": "2021-09-10T23:02:22.258Z", "isPending": false, "isSitemapsIndex": false, "type": "sitemap", "warnings": "0", "errors": "0", "contents": [{"type": "web", "submitted": "5690", "indexed": "0"}]}, "emitted_at": 1674600073798}
+{"stream": "search_analytics_by_date", "data": {"ctr": 0.07940251572327044, "position": 31.88456498951782, "search_type": "web", "date": "2021-09-14"}, "emitted_at": 1674600074364}
```

Comment:
This data seems vague enough to be OK 👍.  Can you confirm that this is not data about www.airbute.com/www.airbute.io?